playbook-website/app/views/pages/visual_guidelines_partials/_positioning.html.erb

```diff
@@ -0,0 +1,43 @@
+<br />
+<br />
+
+<%= pb_rails("title", props: {text: "Positioning", size: 1 }) %>
+<br />
+If you're going to be using position absolute you might find it handy to use some of the index properties. We have multiple ways to use z-index, take a look at the examples below.
+<br />
+<%= render :partial => "pages/visual_guidelines_partials/pb_doc_positioning",
+  locals:{ title: "Global Props" } %>
+  Available in every kit.  These are added globally as they are most flexible when developing.
+  <br />
```

Comment:
1. Lets wrap those plane text areas in body kits
2. Remove break tags and replace with margin/padding props